sdk/confidentialledger/Azure.Security.ConfidentialLedger/README.md

```diff
@@ -126,31 +126,29 @@ Response postResponse = ledgerClient.PostLedgerEntry(
 waitUntil: WaitUntil.Completed,
     RequestContent.Create(
         new { contents = "Hello world!" }));
-string transactionId = postOperation.Id;
-string collectionId = "collection:0";
 
-// Provide both the transactionId and collectionId.
-Response getByCollectionResponse = ledgerClient.GetLedgerEntry(transactionId, collectionId);
+var content = postOperation.GetRawResponse().Content.ToString();
```

Comment:
Should we use `string` here for consistency?

Reply:
Weird - that change exists in this commit, but not in the PR.. 

https://github.com/Azure/azure-sdk-for-net/pull/30448/commits/274315743e157ab7efd36e6ead61f650b848e88f

Reply:
ah - forgot to regen